src/App.js

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
+import {useState} from 'react'
 
 const App = () => {
+  
+  const[textMessage, setTextMessage] = useState(chatMessages)
+  const likedMessage = (id) => {
+  const isliked = textMessage.map((text) =>{
+    if (text.id === id){
+      return {...text, liked: !text.liked}
+    } else{
+      return text;
+    }
+  })
+    setTextMessage(isliked);
+  }
+
+  const totalLikes = (data)=> {
+    const newList = data.filter((ele)=>ele.liked===true)
+    return newList.length
+  }
+
+  const likedCount = totalLikes(textMessage)  
```

Comment:
Like my comment above in `totalLikes()`, you don't need to pass in an argument since `textMessage` is in scope here and we can directly access in the method without passing it in.